pkg/kubelet/metrics/metrics.go

```diff
@@ -126,8 +127,30 @@ const (
 	EphemeralContainer = "ephemeral_container"
 )
 
+type imageSizeBucket struct {
+	lowerBoundInBytes uint64
+	label             string
+}
+
 var (
 	podStartupDurationBuckets = []float64{0.5, 1, 2, 3, 4, 5, 6, 8, 10, 20, 30, 45, 60, 120, 180, 240, 300, 360, 480, 600, 900, 1200, 1800, 2700, 3600}
+	imagePullDurationBuckets  = []float64{1, 5, 10, 20, 30, 60, 120, 180, 240, 300, 360, 480, 600, 900, 1200, 1800, 2700, 3600}
+	// imageSizeBuckets has the labels to be associated with image_pull_duration_seconds metric. For example, if the size of
+	// an image pulled is between 1GB and 5GB, the label will be "1GB-5GB".
+	imageSizeBuckets = []imageSizeBucket{
```

Comment:
can we have N/A bucket for the case when GetImageSize returned unknown? Or this will be zero?

Perhaps `0` can be named `N/A` and everything non zero 0-10MB?

Reply:
If GetImageSize returns error, we will use a empty string here: https://github.com/kubernetes/kubernetes/blob/11cd2ffc0806c8092dcda8995205739551b6d9f0/pkg/kubelet/metrics/metrics.go#L973

Reply:
there is a case when image was not found in cache and 0 returned with the nil error

Reply:
Gotcha, added a N/A tag